samples/bluetooth/broadcast_audio_sink/Kconfig

```diff
@@ -35,4 +35,12 @@ config TARGET_BROADCAST_NAME
 	  Name of target broadcast device. If not empty string, sink device
 	  will only listen to the specified broadcast source. Not case sensitive.
 
+config ENABLE_LC3
```

Comment:
I wonder if we should consider a naming convention for app/sample Kconfig options like prefixing `SAMPLE_` or `APP_` since we do have quite a few Kconfigs (like `LIBLC3` and `FPU`) that doesn't have a prefix, to avoid naming clashes in the future. 

It wouldn't be too far fetched to imagine a sample accidentally using a Kconfig option name that already exist (and there is clearly nothing in Zephyr that catches that (per design I imagine)). 

Not something that needs to be done in this PR, but something we may want to consider going forward.

Reply:
If you propose something I can change it, otherwise I let it be :)